From 94f9f2046a37a8d9446864d9f12ae13235e8d4d4 Mon Sep 17 00:00:00 2001
From: Gleb Mazovetskiy <glex.spb@gmail.com>
Date: Wed, 1 Apr 2020 22:43:42 +0100
Subject: [PATCH] Use EGL instead of OpenGL and GLX

---
 CMakeLists.txt | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0678297a..9dd29b9f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -137,9 +137,6 @@ ENDIF (APPLE)
 
 
 # Find OpenGL
-if (NOT APPLE)
-  find_package(OpenGL REQUIRED)
-endif()
 #find_package(GLM REQUIRED)
 
 if (NOT WIN32)
@@ -325,7 +322,6 @@ else()
     target_link_libraries(barony ${EXECINFO_LIB})
   endif()
 endif()
-target_link_libraries(barony ${OPENGL_LIBRARIES})
 target_link_libraries(barony ${THREADS_LIBRARIES})
 target_link_libraries(barony -lm -lc)
 if( NOT WIN32 AND NOT APPLE)
-- 
2.20.1

